backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/DockerRabbitMQ.java

```diff
@@ -66,6 +66,7 @@
     private final String nodeName;
     private final String rabbitHostName;
     private final String hostNameSuffix;
+    private boolean paused;
```

Comment:
Following discussion with matthieu, AtomicBoolean + synchronized block should rather be used.

Reply:
I though that one https://github.com/linagora/james-project/pull/2420#discussion_r292315106

Reply:
Cool I agree with myself then! Let's keep it as is!